### ISSUE TITLE
learn many to one model

### ISSUE THREAD

**Issue by junyango** [llmhyy/tls_attack#7]
training with negative samples and integrating of loss function for many to 1 model. learn how to implement in many to many model

**Comment by junyango** [comment]
successfully ran the training set with reciprocal normalization; reported 0.999 acc with after convergence in 3 epochs
